src/ontology/uberon-edit.obo

```diff
@@ -224667,6 +224685,71 @@ is_a: UBERON:0002553 ! anatomical cavity
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-07-05T11:30:55Z xsd:dateTime
 
+[Term]
+id: UBERON:8480023
+name: skin of lateral lumbar region of abdomen
+def: "A zone of skin that is part of the lateral lumbar region of abdomen." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of flank" EXACT []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0000180 ! lateral lumbar region of abdomen
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:11:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8480024
+name: skin of sacral region
+def: "A zone of skin that is part of the sacral region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0005473 ! sacral region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:14:27Z xsd:dateTime
+
+[Term]
+id: UBERON:8480025
+name: skin of clavicle region
+def: "A zone of skin that is above the clavicle zone in the pectoral girdle region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0001421 ! pectoral girdle region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:17:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8480026
+name: skin of iliac crest region
+def: "A zone of skin that is above the iliac crest in the pelvic girdle bone/zone." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of posterior iliac crest region" BROAD []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0007830 ! pelvic girdle bone/zone
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:19:16Z xsd:dateTime
+
+[Term]
+id: UBERON:8480027
+name: temple
+def: "The region where the frontal, parietal, temporal, and sphenoid bones fuse. It is located on the side of the head behind the eye and between the forehead and the ear." [https://en.wikipedia.org/wiki/Temple_(anatomy)]
```

Comment:
Consider "posterior" in place of "behind".

Reply:
@bvarner-ebi OK, I will correct to "posterior". 
@bvarner-ebi what do you mean with "EDIT: but not sure if this holds true in all species."?

Reply:
@paulacalipho, it was just a note that I made changes to my original comment. I edited the comment again so it is clearer.

Reply:
@bvarner-ebi Do you think that this definition is better? "The region where the frontal, parietal, temporal, and sphenoid bones fuse."

Reply:
I think that is sufficient. If the second sentence in your original definition is always true, you can keep it as well.

Reply:
@bvarner-ebi I will keep only the first part, to be in the safe side, thanks for the input

Reply:
@bvarner-ebi I am trying to re-open the branch in protege to correct the definition but I got this error message: 
![image](https://user-images.githubusercontent.com/4089751/188113512-14ab4786-ea43-404c-a47c-f3cf535b077a.png)

Reply:
could you run the following in your command line and paste it here please: 
`head uberon-edit.obo`

Reply:
@shawntanzk 
![image](https://user-images.githubusercontent.com/4089751/188119661-688d4ed7-20d9-432f-a534-14468536574c.png)

Reply:
oh ok, im not sure how windows work lol - maybe just open up uberon-edit.obo in a text editor and copy and paste the first few lines here

Reply:
@shawntanzk these are few lines:
format-version: 1.2
subsetdef: added_for_HCA "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
subsetdef: cumbo "CUMBO"
subsetdef: cyclostome_subset "cyclostome structures"
subsetdef: defined_by_cytoarchitecture "brain region defined by cyto-architecture"

and then this is what I see in GitHub desktop
![image](https://user-images.githubusercontent.com/4089751/188120933-5c23826c-fc1c-4edc-8281-5a4831b45751.png)

Reply:
Oh! you have clashes. 
For now you can just remove all the added lines and it should work :)

Reply:
@shawntanzk it works now, many thanks